09/26/2022 02:02:48 PM - validate_responses.py - INFO - Execution begins {current_dir:/scripts/aida/python, script_name:validate_responses.py, arguments:--log /score/logs/validate-responses.log /scripts/aida/python/input/aux_data/log_specifications.txt /data/AUX-data/encoding_modality.txt /data/AUX-data/LDC2021E11.coredocs-xx.txt /data/AUX-data/LDC2021E11.parent_children.tsv /data/AUX-data/LDC2021E11.sentence_boundaries.txt /data/AUX-data/LDC2021E11.image_boundaries.txt /data/AUX-data/LDC2021E11.keyframe_boundaries.txt /data/AUX-data/LDC2021E11.video_boundaries.txt example-task1-run /score/SPARQL-CLEAN-output /score/SPARQL-VALID-output}
09/26/2022 02:02:48 PM - validate_responses.py - INFO - NO_CLASS_NAME - DEFAULT_INFO - validation started
09/26/2022 02:02:51 PM - validate_responses.py - WARNING - Validator - SPAN_OFF_BOUNDARY_CORRECTED - Correcting span '(0,0)-(0,0)' to '(1,0)-(1404,0)' because it is outside the boundary '(1,0)-(1404,0)' of document element 'L0C04CVM8' at /score/SPARQL-CLEAN-output/L0C04CTDS.ttl/AIDA_P3_TA1_CM_A0001.rq.tsv:2
09/26/2022 02:02:51 PM - validate_responses.py - WARNING - Validator - SPAN_OFF_BOUNDARY_CORRECTED - Correcting span '(0,0)-(0,0)' to '(1,0)-(5148,0)' because it is outside the boundary '(1,0)-(5148,0)' of document element 'L0C04CVN2' at /score/SPARQL-CLEAN-output/L0C04CTVE.ttl/AIDA_P3_TA1_CM_A0001.rq.tsv:2
09/26/2022 02:02:51 PM - validate_responses.py - INFO - NO_CLASS_NAME - DEFAULT_INFO - validation finished (warnings:2, errors:0)
